api/src/main/java/marquez/db/OpenLineageDao.java

```diff
@@ -81,6 +85,9 @@ void createLineageEvent(
       PGobject event,
       String producer);
 
+  @SqlQuery("SELECT event FROM lineage_events WHERE run_uuid = :runUuid")
+  List<LineageEvent> listLineageEventsByRunUuid(UUID runUuid);
```

Comment:
_Minor_: For DAOs, we use `findX`, so I would name this method `findOlEventsByRunUuid()`

Reply:
Thank you for the feedback. I'll rename it.